sonar-project.properties

```diff
@@ -19,11 +19,15 @@ sonar.javascript.lcov.reportPaths=web/coverage/lcov.info
 # api/alembic/** - database migrations shouldn't have unit tests.
 # api/app/tests/** - test code doesn't have tests.
 # api/app/db/crud.py - unit tests don't talk to a database.
+# api/app/db/database.py - this should mostly be stubbed out, and not ever be covered by unit tests.
 # web - we don't need coverage reports on stories or tests
+# web/src/features/cHaines/** - the UI is alpha, if we're going to keep it, it needs a massive re-factor.
```

Comment:
I don't think this comment still applies, does it?

Reply:
This section document the reason behind excluding code coverage on some parts of the code - so it will remain relevant until such time as the c-haines front end code has been cleaned up.